subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -515,7 +533,7 @@ uint8_t ll_rx_get(void **node_rx, uint16_t *handle)
 			 */
 			if (rx->type == NODE_RX_TYPE_DC_PDU_RELEASE) {
 				(void)memq_dequeue(memq_ll_rx.tail,
-						   &memq_ll_rx.head, NULL);
+					&memq_ll_rx.head, NULL);
```

Comment:
revert indentation changes.

Reply:
Fixed. I though leading spaces was not allowed, that's why I changed it.  (I guess spaces that can not be replaced with tab are allowed, i.e., less than 8 spaces.)
Fixed.